elasticmq.conf

```diff
@@ -0,0 +1,7 @@
+# the include should be done only once, at the beginning of the custom configuration file
+include classpath("application.conf")
+
+queues {
+  request-general { }
+  omega-editorial-web-application-instance-1 { }
+}
```

Comment:
We should add a docker-compose.yml file to this project as the aim is for it to become self-contained.